deploy/09-mcusd-mceur-metafarm/index.ts

```diff
@@ -0,0 +1,58 @@
+import { DeployerFn } from "@ubeswap/hardhat-celo"
+import { StakingRewards__factory } from "../../typechain"
+import { MoolaStakingRewards__factory } from "../../typechain"
+import { ContractTransaction } from "ethers"
+
+const OWNER_ADDRESS = "0x642abB1237009956BB67d0B174337D76F0455EDd" // Temporary owner, should change to multi-sig
+const REWARDS_DISTRIBUTION_ADDRESS = OWNER_ADDRESS
+const STAKING_TOKEN_ADDRESS = '0x25938830fbd7619bf6cfcfdf5c37a22ab15a93ca' // mcUSD / mcUSD-mcEUR-FP liquidity pool
+
+const MOBI_ADDRESS = "0x73a210637f6F6B7005512677Ba6B3C96bb4AA44B" // MOBI
+const CELO_ADDRESS = "0x471ece3750da237f93b8e339c536989b8978a438" // CELO
+
+export const doTx = async (
+  action: string,
+  tx: Promise<ContractTransaction>
+): Promise<void> => {
+  console.log(`Performing ${action}...`);
+  const result = await (await tx).wait();
+  console.log(`${action} done at tx ${result.transactionHash}`);
+};
+
+const main: DeployerFn<{}> = async ({
+  deployCreate2,
```

Comment:
nit: weird parameter name

Reply:
This is the same name used by other deployer functions; it's passed in using the `deployCreate2` key since we're using array destructuring here.. I believe it comes from `makeDeployTask` in the `@ubeswap/hardhat-celo` package.